doc/manual/src/release-notes/rl-next.md

```diff
@@ -20,3 +20,10 @@
   to the embedded store and not to the host's Nix store.
 
   This requires the `discard-references` experimental feature.
+
+* Fix misread of source if path is already valid.
+  When receiving a stream of NARs through the ssh-ng protocol, an already
+  existing path would cause the NAR archive to not be read in the stream,
+  resulting in trying to parse the NAR as a ValidPathInfo. This results in
+  the error message:
+      error: not an absolute path: 'nix-archive-1'
```

Comment:
```suggestion
```

since we don't list bug fixes in the release notes.

Reply:
@edolstra: Oh, but why? Except it's more work. I think long-term, large-scale users with delicate upgrade requirements may benefit from this.